lib/chef/knife/ec2_server_create.rb

```diff
@@ -453,6 +453,10 @@ class Ec2ServerCreate < Knife
         :boolean => true,
         :default => false
 
+      option :instance_initiated_shutdown_behavior,
+        :long => "--instance-initiated-shutdown-behavior",
```

Comment:
The option definition doesn't look correct. It looks like this option should accept a string value. For that the `:long` attribute should be given like this:
```
:long => "--instance-initiated-shutdown-behavior STUTDOWN_BEHAVIOR"
```
Otherwise a boolean value will be passed for any input.